esm/html/document.js

```diff
@@ -77,8 +77,7 @@ export class HTMLDocument extends Document {
    */
   get title() {
     const {head} = this;
-    let title = head.getElementsByTagName('title')[0];
-    return title ? title.textContent : '';
+    return head.getElementsByTagName('title')[0]?.textContent || '';
```

Comment:
`.at(0)` ... not `[0]` ... `[0]` bails out performance if no `title` exists, `.at(0)` doesn't suffer this issue.